gettingStartedWithAstraBlock/README.md

```diff
@@ -12,9 +12,30 @@ Let's get started!
 None
 
 ## 1 Introduction to Astra Block
-With Astra Block, you can stream real-time blockchain data directly to your Astra database. Simply enable the feature within Astra, create a blockchain database and start receiving data.
+With Astra Block, you can stream real-time blockchain data directly to your Astra database. Simply request access to the feature within Astra (we'll enable it for your account within 24 hours) and you can instantly create a blockchain database that auto updates as new blocks are mined.
+
+We currently support Ethereum, with Polygon and Bitcoin coming soon. Interested in support for other chains? [Let us know](mailto:blockchain@datastax.com).
+
+We currently support the following data models:
+- Blocks
+- Contracts
+- Transactions
+- NFTs
+- Tokens
+- Token Transfers
+- Traces
+- Logs
+- Dashboard Analytics
+- Block numbers by date
+- Event signatures
+- Assets
+
+Interested in support for other data? [Let us know](mailto:blockchain@datastax.com).
+
+Full schema can be viewed in [our docs](https://docs.datastax.com/en/astra-serverless/docs/block/overview.html).
```

Comment:
Unless we plan on updating the content in this doc link we should not include it in this guide.. 

Also There is a ton of good example queries in that guide, have you thought of scrapping that docs link and moving that info to this guide instead?